doc/whatsnew/2.6.rst

```diff
@@ -14,6 +14,8 @@ Summary -- Release highlights
 New checkers
 ============
 
+* Add `nan-comparison` check for comparison of NaN values
```

Comment:
Could you rebase and move that to `2.7.rst`, please ? Sory it's due to #4095 being merged.

Reply:
Okay sure